### ISSUE TITLE
<Space> and <Shift-Space> not documented for normal mode

### ISSUE THREAD

**Issue by prosoitos** [qutebrowser/qutebrowser#4845]
Unless I missed something, `<Space>` and `<Shift-Space>` are missing from the documentation in normal mode. They don't show anywhere in https://www.qutebrowser.org/doc/help/settings.html#bindings.default.

It took me a while to figure out what exactly they trigger. I eventually figured out that it is `scroll page-down` and `scroll page-up`. But since there is also the command `scroll-page`, without the documentation, I was stuck with fiddling with countless variations of `scroll-page 0 1` when the behaviour I was after was `scroll page-down` (in order to remap it).

**Comment by user202729** [comment]
No, that's a chromium/qtwebengine feature, the space is just passed to the page. Observe that it adds a space when the cursor is in a text area, and space has the same behavior in chrome.

**Comment by prosoitos** [comment]
Oh, I see. I can't see the difference between `<space>` and `scroll page-down` though.

What I really wanted to remap in fact was Shift-Space. I tried using `:fake-key <Shift-Space>`, but that didn't work (I got an error "Shift-Space is not a valid command"). Maybe, if it is a qtwebengine thing, it has to be written `:fake-key <s-space>` or `:fake-key <shift-space>` or some other variation instead of `:fake-key <Shift-Space>`??

That said, mapping `scroll page-up` instead was fine since I can't see the difference.

**Comment by Jan-PieterBaert** [comment]
`:fake-key <shift-space>` seems to work for me

**Comment by The-Compiler** [comment]
There are other keys which are handled in QtWebEngine in this way (for example cursor keys). I don't think we really can document them all, but I agree it's a bit weird that they exist. It certainly would be more consistent to only have our own bindings and not have keys passed through to Qt at all. I'll need to think some more about it, but not right now.

**Comment by prosoitos** [comment]
At least, things make a lot more sense to me now. So thank you!!
